.github/workflows/tests.yml

```
@@ -0,0 +1,34 @@
+name: Tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
```

Rather than creating a tech debt ticket, why don't we just use Node.js v22 now?

```suggestion
          node-version: 22
```

---

The actual runner doesn't support node 22 which is why I used node 20 here to match the node runner version

---

Yes it does. See https://github.com/dequelabs/launchpad/blob/develop/.github/actions/dependencies/action.yml#L11-L13

---

@stephenmathieson The `setup-node` action does, but that's not what controls the actual runtime environment, that just controls what we're running our build/tests on. The thing that controls what node version a user will actually be using to *run* the action code is the `runs: { using: 'node20' }` directive in `action.yml`, where `node20` is the only supported option. https://github.com/actions/runner/issues/3600 tracks support for custom actions using `node22` (but I would be surprised if GitHub actually implemented that, historically they've supported every-other-LTS at best)